agimus_controller/problem.py

```diff
@@ -173,26 +115,22 @@ def set_sub_path_running_models(self, path_idx):
                     self.DT,
                 )
             )
-        self.hpp_paths[path_idx].running_models = running_models
+        self.running_models = running_models
```

Comment:
be carefull in python when you do that you do a pointer copy not an actual copy.
Is this what you really intend to do?